### ISSUE TITLE
Error in postgres while TRACY is on

### ISSUE THREAD

**Issue by pitr82** [nextras/orm#339]
Ahoj, 
testuju na postgresql 11.2
a když mám **zapnutou laděnku**, tak mi to v logu postgresu hlásí chyby:

`db_1       | 2019-02-26 12:50:27.485 UTC [110] ERROR:  syntax error at or near "SET" at character 9
db_1       | 2019-02-26 12:50:27.485 UTC [110] STATEMENT:  EXPLAIN SET TIME ZONE 'Europe/Prague'
db_1       | 2019-02-26 12:50:27.488 UTC [110] ERROR:  syntax error at or near "START" at character 9
db_1       | 2019-02-26 12:50:27.488 UTC [110] STATEMENT:  EXPLAIN START TRANSACTION
db_1       | 2019-02-26 12:50:27.491 UTC [110] ERROR:  syntax error at or near "SAVEPOINT" at character 9
db_1       | 2019-02-26 12:50:27.491 UTC [110] STATEMENT:  EXPLAIN SAVEPOINT "NEXTRAS_SAVEPOINT_2"
db_1       | 2019-02-26 12:50:27.493 UTC [110] ERROR:  syntax error at or near "RELEASE" at character 9
db_1       | 2019-02-26 12:50:27.493 UTC [110] STATEMENT:  EXPLAIN RELEASE SAVEPOINT "NEXTRAS_SAVEPOINT_2"
db_1       | 2019-02-26 12:50:27.493 UTC [110] ERROR:  syntax error at or near "ROLLBACK" at character 9
db_1       | 2019-02-26 12:50:27.493 UTC [110] STATEMENT:  EXPLAIN ROLLBACK
db_1       | 2019-02-26 12:56:16.724 UTC [130] ERROR:  syntax error at or near "SET" at character 9
db_1       | 2019-02-26 12:56:16.724 UTC [130] STATEMENT:  EXPLAIN SET TIME ZONE 'Europe/Prague'
db_1       | 2019-02-26 12:56:16.727 UTC [130] ERROR:  syntax error at or near "START" at character 9
db_1       | 2019-02-26 12:56:16.727 UTC [130] STATEMENT:  EXPLAIN START TRANSACTION
db_1       | 2019-02-26 12:56:16.730 UTC [130] ERROR:  syntax error at or near "SAVEPOINT" at character 9
db_1       | 2019-02-26 12:56:16.730 UTC [130] STATEMENT:  EXPLAIN SAVEPOINT "NEXTRAS_SAVEPOINT_2"
db_1       | 2019-02-26 12:56:16.732 UTC [130] ERROR:  syntax error at or near "RELEASE" at character 9
db_1       | 2019-02-26 12:56:16.732 UTC [130] STATEMENT:  EXPLAIN RELEASE SAVEPOINT "NEXTRAS_SAVEPOINT_2"
db_1       | 2019-02-26 12:56:16.732 UTC [130] ERROR:  syntax error at or near "COMMIT" at character 9
db_1       | 2019-02-26 12:56:16.732 UTC [130] STATEMENT:  EXPLAIN COMMIT`

Tady je výpis z TRACY

ms | SQL query
-- | --
1.95 | SET TIME ZONE 'Europe/Prague'copy
2.23 | SELECT "alarm".* FROM "alarm" AS "alarm" WHERE ("alarm"."acknowledge" = FALSE)1 row copy explain
1.11 | SELECT "location".* FROM "location" AS "location" WHERE ("location"."id" = 1)1 row copy explain
0.16 | START TRANSACTIONcopy
0.71 | SELECT "port".* FROM "port" AS "port" WHERE ("port"."id" = 1)1 row copy explain
0.76 | SELECT "event".* FROM "event" AS "event" WHERE ("event"."id" = 1)1 row copy explain
0.13 | SAVEPOINT "NEXTRAS_SAVEPOINT_2"copy
2.11 | INSERT INTO "alarm" ("location_id", "port_id", "event_id", "from", "to", "acknowledge", "email_sent", "created_at") VALUES (1, 1, 1, NULL, NULL, TRUE, NULL, '2019-02-26 13:56:16.472500'::timestamptz)0 rows copy explain
0.37 | SELECT CURRVAL('alarms_id_seq')1 row copy explain
0.17 | RELEASE SAVEPOINT "NEXTRAS_SAVEPOINT_2"copy
3.06 | COMMITcopy

**Comment by hrach** [comment]
No, on se explain pousti pro kazdy dotaz, protoze neni moc jak poznat, jestli pro nej DB podporuje explain. Ze to zpusobi nekde zalogovani me nenapadlo 🤓 Vetsinou to navic pouziva clovek lokalne, kde error log moc neresi.
